### DISCUSSION TITLE
Group: nested

### DISCUSSION THREAD

**Discussion by elfeffe** [amiranagram/livewire-sortablejs#4] [Ideas]
I need to use the nested option, as in sortablejs example.
Do you think you can separate your code from the sortablejs code, so we can instantiate with the needed settings?
I think you should separate your code from sortablejs one anyway, because one can already have the library installed.

**Comment by amiranagram** [comment]
Hi, I'm currently working on other options, nested Including. 

The only reason I merged my package with SortableJS was to be able to use it easily from CDN. I don't know about separation, I'll have to think about it. I don't wanna break people's code.

You can use livewire-sortablejs in your bundle separately tho. Just import the unbundled source:
```
require('livewire-sortablejs/src/index')
```
